browser/brave_content_browser_client_browsertest.cc

```diff
@@ -266,6 +266,31 @@ IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, RewriteChromeSync) {
   }
 }
 
+IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, RewriteAdblock) {
```

Comment:
@simonhong Here's the test I've added for re-write of adblock

Reply:
It doesn't run on Android.

Reply:
ok so I won't need build flag here